CHANGELOG.md

```diff
@@ -62,6 +62,12 @@ The default keyboard shortcuts are displayed in the small widget shown when hove
 
 To enable the inline suggestions based on the kernel history, go to Settings → Inline Completer → History provider → check the "enabled" checkbox.
```

Comment:
I can't see `Inline Completer` - am I missing something?

![image](https://github.com/jupyterlab/jupyterlab/assets/881019/b6b1a53d-8d95-41be-8235-797847964ad3)

Reply:
I cannot see code completer section on your screenshot either. Can you choose and reopenn the settings panel? There may be a glitch on intital load .

Reply:
Hmm, settings now looks like:

![image](https://github.com/jupyterlab/jupyterlab/assets/881019/51d6c71f-b38b-4a5b-ad4a-0d4c866cace1)

Reply:
Maybe I updated without closing down my JupyterLab tab? :thinking:

Unless you get flooded with other questions about missing settings , it's probably just user error - sorry for the noise!

Reply:
Sorry, I do not follow - your second screenshot looks correct to me, isn't it?

Reply:
Or is it about jupyter-ai not showing up in there? Which version do you have installed?

Reply:
Yes, the 2nd screenshot looks correct so I'm not sure why it didn't look like that before. I was wondering if I hadn't closed down my JupyterLab tab when installing 4.1.0rc0 and so initially wasn't seeing the updated UI.

After taking a look this morning, after a reboot, it looks fine so I think it's likely user error and not an issue with JupyterLab.

Reply:
I have `jupyter-ai-core` 2.9.1 installed:

![image](https://github.com/jupyterlab/jupyterlab/assets/881019/30252008-7d94-414c-8bfb-bdf4b9903c28)

SO, IIUC I need to wait for 2.10 to be able to use it for completions.